### PR TITLE
Added new user view

### PR DESCRIPTION
**Pull request by ar4s**
User can view assigned equipment. The view is very similar to additional user
info but this view is strongly linked with logged user.
user_equipmentr

### CODE REVIEW

File: src/ralph/accounts/admin.py [hunk 1]
```diff
@@ -61,19 +61,21 @@ def url(self, item):
...
+    def get_user(self):
+        if not self.user:
+            raise NotImplementedError('Please specify user.')
+        return self.user
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         asset_list_queryset = BackOfficeAsset.objects.filter(
-            user=self.object
+            user=self.get_user()
```
Comment (mkurek) [inline src/ralph/accounts/admin.py]: here should be `Q(user=self.get_user()) | Q(owner=self.get_user())`

File: src/ralph/accounts/views.py [hunk 2]
```diff
@@ -1,7 +1,28 @@
...
 class UserProfileView(RalphTemplateView):
-
     template_name = 'ralphuser/user_profile.html'
+
+
+class CurrentUserInfoView(UserInfoMixin, TemplateView):
+    template_name = 'ralphuser/my_equipment.html'
+
+    def dispatch(self, request, *args, **kwargs):
+        if not request.user.is_authenticated():
```
Comment (mkurek) [inline src/ralph/accounts/views.py]: wouldn't it be better to use https://docs.djangoproject.com/en/1.8/topics/auth/default/#the-login-required-decorator ?

**Comment by mkurek** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**